frontend/javascripts/dashboard/registration_form.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+import { Form, Input, Button, Row, Col, Icon, Checkbox } from "antd";
+import React from "react";
+
+import { loginUser } from "admin/admin_rest_api";
+import { setActiveUserAction } from "oxalis/model/actions/user_actions";
+import Request from "libs/request";
+import Store from "oxalis/throttled_store";
+import messages from "messages";
+
+const FormItem = Form.Item;
+const { Password } = Input;
+
+type Props = {|
+  form: Object,
+  onRegistered: true => void,
```

Comment:
I think, the parameter `true` of the `onRegistered` function is never used (otherwise, it probably should be a `boolean`).

Reply:
It is expected by the calling code. This has todo with the shared API of both registration forms.

Reply:
For this version of the registration form, it is always `true` therefore the literal type.